include/gc/Transforms/Utils/StructuredOpMatcher.h

```diff
@@ -217,6 +217,16 @@ template <typename T> struct EqualsTo {
 };
 template <typename T> EqualsTo(T) -> EqualsTo<T>;
 
+// Callable object to check if the input is greater than or equal to specified
+// `value`.
+struct GreaterThanOrEqualTo {
+  GreaterThanOrEqualTo() = delete;
+  explicit GreaterThanOrEqualTo(size_t value) : value(value){};
+  const size_t value;
+
+  bool operator()(size_t value) const { return value >= this->value; }
+};
```

Comment:
`std::greater_equal`?

Reply:
This is also one of a series of Op matcher utils borrowed from TPP, and it requires different signature from std.